server/src/main/java/org/spine3/server/storage/RecordStorage.java

```diff
@@ -92,26 +93,43 @@ protected RecordStorage(boolean multitenant) {
     }
 
     /**
-     * {@inheritDoc}
+     * Writes a record and its
+     * {@linkplain org.spine3.server.entity.storage.StorageFields Storage Fields} into the storage.
```

Comment:
One thought that I have is why we keep calling that `StorageFields`, while we don't have anything named like that in our API? Maybe we should call it `Entity Columns` and reference the `Column` class?